src/main/java/sc/fiji/coloc/results/PDFWriter.java

```diff
@@ -153,8 +226,10 @@ public void handleValue(String name, double value, int decimals) {
 	/**
 	 * Prints an image into the opened PDF.
 	 * @param image The image to print.
+	 * @param string2 
+	 * @param string 
```

Comment:
These two `@param` annotations make the [Travis build](https://travis-ci.org/fiji/Colocalisation_Analysis/builds/399673241) fail (all other messages are merely warnings).

The javadoc should correspond to the parameter names of the method.